src/elements/activityIndicator.js

```diff
@@ -181,10 +181,11 @@ class ActivityIndicator extends React.PureComponent {
             <div
                 className={containerClasses}
                 id={id}
-                style={({
+                style={{
                     ...style,
-                    transform: `scale(${(1 / 68) * size}) ${style && style.transform}`,
-                })}
+                    transform: `scale(${(1 / 68) * size})`,
```

Comment:
Is 1/68 some kind of "magic number"?  Would it be worth declaring a `const` to describe what this value represents?  Not blocking.

Reply:
Fixed.